contracts/KetherNFTLoaner.sol

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import '@openzeppelin/contracts/access/Ownable.sol';
+import '@openzeppelin/contracts/utils/math/SafeMath.sol';
+import 'ketherhomepage/contracts/KetherNFT.sol';
```

Comment:
Could probably cut back a lot of gas if you just make an interface subset for the functions you're using.

Reply:
Updated in https://github.com/moontography/contracts/pull/2/commits/51ab1e36a6cbb5e737ee94364a3e939c0d16dd4f